3sum.java

```diff
@@ -0,0 +1,30 @@
+//time complexity O(n)
```

Comment:
O(n2) is correct time complexity

Reply:
Got it! It makes sense since in worst case scenario, the " i " pointer will iterate over the whole array (until nums.length -2) which is O(n )and the low and high pointer inside contribute to nested O(n) time complexity which makes it O(n^2) time complexity. Thanks for spotting that out.

Reply:
Yuup